src/pages/Home.jsx

```diff
@@ -1,42 +1,71 @@
 import { useDispatch, useSelector } from 'react-redux';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import { useState } from 'react';
 import Form from '../components/Form';
+import 'react-circular-progressbar/dist/styles.css';
 import './home.css';
 import { deleteBook, deleteAPIBook } from '../components/redux/books/booksSlice';
 
 // Import the initialState and display
 export default function Home() {
   const dispatch = useDispatch();
   const books = useSelector((state) => state.book.books);
+  const [percentage, newPercentage] = useState(65);
 
+  const handlePercentage = (e) => {
+    e.preventDefault();
+    newPercentage(Math.floor(Math.random() * 100));
+  };
   return (
     <div className="bookList">
       { (books ?? []).filter((b) => b.item_id).map((book) => (
         <div className="eachBook" key={book.item_id} id={book.item_id}>
           <div className="left-pan">
             <h4>{book.category}</h4>
             {' '}
-            <h1>{book.author}</h1>
+            <h1>{book.title}</h1>
             {' '}
-            <p className="booktitle">{book.title}</p>
+            <p className="booktitle">{book.author}</p>
             {' '}
             <div className="buttonDiv">
               {' '}
               <button type="button">Comments</button>
-              <button className="remove" type="button" onClick={() => { dispatch(deleteBook(book.item_id)); dispatch(deleteAPIBook(book.item_id)); }}>REMOVE</button>
+              <span className="Line-2" />
+              <button className="remove" type="button" onClick={() => { dispatch(deleteBook(book.item_id)); dispatch(deleteAPIBook(book.item_id)); }}>Remove</button>
+              <span className="Line-2" />
               <button type="button">Edit</button>
             </div>
```

Comment:
- [ ] Good job adding the line between the buttons🥇🥇 However, please make sure that it is according to the given Zeplin design. Right now, I can see some white space in between and also it is not aligned according to the design guidelines. The width can be fixed by removing the `width` from `.Line-2`. See the images below for more clarity 💡

Your design 🔽 | Zeplin design 🔽
---| ---|
![image](https://user-images.githubusercontent.com/106119105/230636956-83aa6120-c66a-485c-bf45-f4668560d453.png) | ![image](https://user-images.githubusercontent.com/106119105/230636745-488e9aef-f542-4548-938e-84452e8d6241.png)